scalableminds/webknossos
# Show inactive users warning

This PR adds an Antd Alert when there are new inactive users (<14 days). They can be activated directly from this Banner. The original issue #2928 also shows a delete button, but afaik we don't implement this functionality. Should this be added in your opinion? Otherwise there would be no way to get rid of unwanted users, as activating them would be the only option.

![new-users](https://user-images.githubusercontent.com/1702075/43466476-e51b8b92-94df-11e8-850c-15fdcd598f0c.png)


### URL of deployed dev instance (used for testing):
- https://showinactiveuserswarning.webknossos.xyz

### Steps to test:
- Create one or more new users. Then login again with an admin account.
- Visit the user list and there should be an Alert notifying you of the new inactive users. Activating them should work. Deactivating them again should add them back to the Alert.

### Issues:
- fixes #2928

------
